app/views/hq/partners/show.html.erb

```diff
@@ -1,3 +1,11 @@
+<div class="action_items">
+  <div class="instance">
+    <%= link_to('Edit Partner', edit_hq_partner_path(@partner.id),
+                class: 'btn btn-default', role: 'button') %>
+    <%= link_to('Upload Orphan List', upload_hq_partner_pending_orphan_lists_path(@partner.id),
```

Comment:
The router only allows you to override the current prefix, so
`get 'upload', on: :collection, as: :foobar` is `foobar_hq_partner_pending_orphan_lists_path` instead of `upload_hq_partner_pending_orphan_lists_path`.  The alternative is to hardcode the URL outside of the blocks & namespaces like `:new_sponsorship` is, which I think would be more messy and harder to understand